### PR TITLE
e2e image

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
e2e image with needed binaries

```release-note
NONE
```

### CODE REVIEW

**Comment by xmudrii** [comment]
/hold
until there is no job that will push this image to the registry

File: hack/images/kubeone-e2e/release.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (kron4eg) [reply hack/images/kubeone-e2e/release.sh]: fixed

File: hack/images/kubeone-e2e/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+FROM golang:1.13.4-alpine
```
[the remark this answers is not in the document]
Comment (kron4eg) [reply hack/images/kubeone-e2e/Dockerfile]: fixed

**Comment by kron4eg** [comment]
To unblock the situation with cyclic dependency between this PR and #729 I'll override failed jobs and fix them in #729

/override pull-kubeone-e2e-aws-conformance-1.14
/override pull-kubeone-e2e-aws-conformance-1.15
/override pull-kubeone-e2e-aws-conformance-1.16
/override pull-kubeone-e2e-aws-upgrade-1.14-1.15
/override pull-kubeone-e2e-aws-upgrade-1.15-1.16

**Pull request merged**